src/EditorFeatures/DiagnosticsTestUtilities/Diagnostics/AbstractDiagnosticProviderBasedUserDiagnosticTest.cs

```diff
@@ -109,24 +109,6 @@ public void TestSupportedDiagnosticsMessageDescription()
             }
         }
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/26717")]
```

Comment:
📝 This test would automatically run for all tests in types derived from this class. If we unskip the test, does it pass now?

Reply:
@sharwell On its current state, it doesn't pass. Certain analyzers are known to have no help link (see the first commit).

Even with the first commit, it doesn't pass, but it fails for dummy analyzers used in testing only.

Reply:
A new commit was pushed before the build completed, so I can't see the cases that failed. Do you have any examples?

Reply:
https://github.com/dotnet/roslyn/blob/6da1274c9d24c2f90a48290394a951b23617f2a3/src/EditorFeatures/CSharpTest/Diagnostics/MockDiagnosticAnalyzerTests.cs#L21 is an example. But let me open a draft with only the first commit.

Reply:
@sharwell The build in the draft #51882 is ready.